core/kernel/impl/class.ApiModelOO.php

```diff
@@ -339,37 +339,21 @@ public function getRootClasses()
      */
     public function setStatement($subject, $predicate, $object, $language)
     {
-        $returnValue = (bool) false;
-
-        
-        $dbWrapper 	= core_kernel_classes_DbWrapper::singleton();
-        $platform   = $dbWrapper->getPlatForm();
-        $localNs 	= common_ext_NamespaceManager::singleton()->getLocalNamespace();
-        $mask		= 'yyy[admin,administrators,authors]';	//now it's the default right mode
-        $query = 'INSERT INTO statements (modelid,subject,predicate,object,l_language,author,epoch)
-        			VALUES  (?, ?, ?, ?, ?, ? , ?);';
-
-        try{
-	        $returnValue = $dbWrapper->exec($query, array(
-	       		$localNs->getModelId(),
-	       		$subject,
-	       		$predicate,
-	       		$object,
-	       		$language,
-	       		\common_session_SessionManager::getSession()->getUserUri(),
-	            $platform->getNowExpression()
-	             
-	        ));
-        }
-        catch (DBALException $e){
-	        if($e->getCode() !== '00000'){
-				throw new common_Exception ("Unable to setStatement (SPO) {$subject}, {$predicate}, {$object} : " . $e->getMessage());
-			}
-        }
-        
-        
-
-        return (bool) $returnValue;
+        $localNsManager = common_ext_NamespaceManager::singleton();

```

Comment:
I can see 2 different styles of using an object
common_ext_NamespaceManager
and 
\common_session_SessionManager
and I would recommend sticking to one or another for consistency